docs/policies.rst

```diff
@@ -196,6 +196,19 @@ training data. It predicts the next action with confidence ``1.0``
 if this exact conversation exists in the training data, otherwise it
 predicts ``None`` with confidence ``0.0``.
 
+Mapping Policy
+^^^^^^^^^^^^^^^^^^
+
+The ``MappingPolicy`` can be used to directly map intents to actions such that
+the mapped action will always be executed. The mappins are assigned by giving
+and intent the property `'triggers'`, e.g.:
+
+.. code-block:: yaml
+
+  intents:
+   - greet: {triggers: utter_goodbye}
+
+An intent can only be mapped to at most one action.
```

Comment:
Human: Hello
Bot: Bye

I know it's just a doc example, but surely `greet` should trigger `utter_greet`, no? 😄

Reply:
We could change the policy to execute ActionBack after it executed a mapped action. 
But that might mes up the tracker if the mapping is in the examples right? So we have to somehow make this clear in the docs.

Reply:
in the latest commit I did just that, executing action_back after if the last action is the action that the current intent in mapped to

Reply:
no no no i don't think the policy should execute ActionBack after it executed a mapped action. I'm sure we discussed this, that's going to cause some undesired behaviour

Reply:
We should explain the `UserUtteranceReverted` thing in the docs though

Reply:
Would it not be better to have the policy append `UserUtteranceReverted`? Otherwise mapped actions could not be normal actions and vice versa

Reply:
Actually that wouldn't work because the action is executed only after any policy methods run.
But then how can we make the action return `UserUtteranceReverted` if it was performed based on the mapping policy?

Reply:
@imLew this is up to the user to implement in their own action.

Reply:
That would mean that if users want to map to an already existing action they would have to create a copy that also returns UserUtteranceReverted.
If it always needs to be done anyway I think we should have the policy do it, maybe by just having an action that returns only one `UserUtteranceRevert`.

Reply:
I'm not sure I agree, there may be cases where people want to change that behaviour. @Ghostvv what are your thoughts

Reply:
I don't see why we need this now. I think we should not implement any revert behaviour for all mapped actions. This will be working just fine for the use cases we need it for atm (e.g. restart and back) and we can change this once we discover that it really is not working for other actions.

Reply:
I can see both arguments. I agree it is not a necessity. I guess if we want it, we could create action wrapper in the mapping policy, that when passed mapping actions, creates a new action with appended `UserUtteranceReverted()`. The problem with that, is all events returned by original action will be forgotten, so no slots would be set. This wrapper could insert  `UserUtteranceReverted()` in the beginning though

Reply:
Yeah let's not implement the revert behaviour for now, but just add a note to the docs about it